src/jit/gentree.h

```diff
@@ -149,6 +149,18 @@ struct InlineCandidateInfo;
 
 typedef unsigned short AssertionIndex;
 
+//------------------------------------------------------------------------
+// GetAssertionIndex: return 1-based AssertionIndex from 0-based int index.
```

Comment:
Why do we do this? Is it to pack Assertion Numbers in the assertion bitmaps? Thus, bit 0 == assertion# 1, bit 1 == assertion # 2, etc.? We should stop doing this and just "burn" bit 0, so we don't need to do this math. Thus, bit0 == unused, bit 1 == assertion# 1, bit 2 == assertion #2, etc. We do it this way for BasicBlock numbers already. It's simpler and doesn't require this logic. There will possibly be diffs as one fewer assertion would be allowed in the bitmaps (assuming those are fixed sized)

---
In reply to: [122082167](https://github.com/dotnet/coreclr/pull/11945#discussion_r122082167) [](ancestors = 122082167)